deployments/rialto/Bridge.Dockerfile

```diff
@@ -81,5 +81,7 @@ ENV HEALTH=$HEALTH
 HEALTHCHECK --interval=2m --timeout=10s \
   CMD curl -f $HEALTH || exit 1
 
+EXPOSE 9944 9933
```

Comment:
FYI, this might not do what you think it does. The `EXPOSE` command is [more for documentation](https://docs.docker.com/engine/reference/builder/#expose), and doesn't actually expose the ports for you.

Reply:
Oh, thanks for the info. Didn't know that.
```suggestion
```